src/static/js/components/opinionOnboarding/view.js

```diff
@@ -0,0 +1,56 @@
+/*global $ events getCookie setCookie */
+
+var OpinionOnboardingView = function() {};
+
+OpinionOnboardingView.prototype.initEvents = function(opinionModalView) {
+  this.opinionOnboardingElement = $('.js-opinionOnboarding');
+  this.buttonElement = $('.js-opinionOnboarding .js-button');
+  this.cookieName = 'opinionOnboardingCookie';
+  this.opinionModalView = opinionModalView;
+  this.subscribers();
+  this.publishers();
+};
+
+OpinionOnboardingView.prototype.publishers = function() {
+  var self = this;
+
+  var cookie = getCookie(this.cookieName);
+  if (self.opinionOnboardingElement.data('isAuthenticated')) {
+    if (!cookie) {
+      events.showOpinionOnboarding.publish();
+    } else {
+      if (self.opinionModalView.hasActiveOpinionCards()) {
+        events.openOpinionModal.publish(null);
+      }
```

Comment:
If there is no opinion cards, should not trigger `events.showTextSelectionOnboarding`?